pkg/kbuser/object_ids.go

```diff
@@ -24,32 +24,55 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// If in the future a need arises to analyze both cluster-scoped and namespace-scoped objects, switch to using
```

Comment:
I would put this note _after_ the live code rather than _before_, expecting readers to be more interested in the live code.

Reply:
done

Reply:
Good.